test/nnstreamer/test_nnstreamer_single.cpp

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2021 Jihoon Lee <jhoon.it.lee@samsung.com>
+ *
+ * @file test_nnstreamer_single.cpp
+ * @date 10 November 2021
+ * @brief NNTrainer filter integrated test with nnstreamer
+ * @see	https://github.com/nnstreamer/nntrainer
+ * @author Jihoon Lee <jhoon.it.lee@samsung.com>
+ * @bug No known bugs except for NYI items
+ */
+
+#include <gtest/gtest.h>
+
+#include <nnstreamer-single.h>
+#include <nnstreamer.h>
+
+#include <nntrainer_test_util.h>
+
+static std::string mnist_model_path =
+  getResPath("mnist.ini", {"test", "test_models", "models"});
+
+TEST(mlInference, singleshot_p) {
```

Comment:
add negative cases. (you need to keep the number of negative > the number of positive)

Reply:
updated :)